lib/archethic_web/graphql_schema/beacon_chain_summary_type.ex

```diff
@@ -0,0 +1,87 @@
+defmodule ArchethicWeb.GraphQLSchema.BeaconChainSummary do
+  @moduledoc false
+
+  use Absinthe.Schema.Notation
+  alias Archethic.BeaconChain.SummaryAggregate
+
+  @desc """
+  [Beacon Chain Summary] represents the beacon chain aggregate for a certain date
+  """
+
+  @default_limit 100
+
+  object :beacon_chain_summary do
+    field(:version, :integer)
+    field(:summary_time, :string)
+    field(:availability_adding_time, list_of(:integer))
+    field(:p2p_availabilities, :p2p_availabilities)
+
+    field(:transaction_summaries, list_of(:transaction_summary)) do
+      arg(:paging_offset, :non_neg_integer)
+      arg(:limit, :pos_integer)
+
+      resolve(fn args,
+                 %{
+                   source: %SummaryAggregate{
+                     transaction_summaries: transaction_summaries
+                   }
+                 } ->
+        limit = Map.get(args, :limit, @default_limit)
+        paging_offset = Map.get(args, :paging_offset, 0)
+
+        result =
+          transaction_summaries
+          |> Enum.drop(paging_offset)
+          |> Enum.take(limit)
+
+        {:ok, result}
+      end)
+    end
+  end
+
+  @desc """
+  [Transaction Summary] Represents transaction header or extract to summarize it
+  """
+  object :transaction_summary do
+    field(:timestamp, :timestamp)
+    field(:address, :address)
+    field(:movements_addresses, list_of(:address))
+    field(:type, :string)
+    field(:fee, :integer)
+  end
+
+  scalar :p2p_availabilities do
```

Comment:
For the p2p availabilities, I would suggest to change the format of the returned data.
Actually the p2p availabilities are understanble by the nodes, but for a developer or a user, having the subset and a list of 0/1 without knowing what is the node behind is not very meaningfull.
Maybe format the availabilitie to something like this ?
```jsonc
{
  "p2pAvailabilities": [
    {
      "public_key": "0123456798ABCDEF",
      "available": true, // false
      "avairage_availability": 0.89,
      "endOfNodeSynchronization": false // true
    },
    {
      "public_key": "0123456798ABCDEF",
      "available": true, // false
      "avairage_availability": 0.89,
      "endOfNodeSynchronization": false // true
    }
  ]
}
```